File: python/Vowel.py
# def isVowel(str):
#     vowel_count = 0
#     consonant_count = 0
#     vowel = "aeiou"
#     for char in str.lower():
#         if char in vowel:
#             vowel_count += 1
#         elif char.isalpha():
#             consonant_count += 1
#     return vowel_count, consonant_count

# user_string = input("Enter text :: ")
# v, w = isVowel(user_string)
# print(f"Vowel: {v} and consonant : {w}");


def isVowel(str):
    vowel_count = 0
    consoant_count = 0
    vowel = "aeiou"
    for char in str:
        if char in vowel:
            vowel_count += 1
        elif char.isalpha():
            consoant_count += 1
    return vowel_count, consoant_count

user_input = input("Enter text :: ")
v, c = isVowel(user_input.lower())
print(f"Vowel: {v} and Consonant: {c}")

File: Java/Vowel.java

public class Vowel {
    public static void main(String[] args) {
        System.out.print("Enter text :: ");
        String userInput = System.console().readLine();
        int[] counts = isVowel(userInput.toLowerCase());
        int vowelCount = counts[0];
        int consonantCount = counts[1];

        System.out.println("Vowel: "+vowelCount+" and Consonant: "+consonantCount);

    }

    public static int[] isVowel(String str) {
        int vowelCount = 0, consonantCount = 0;
        String vowels = "aeiou";

        for(char ch : str.toCharArray()) {
            if(vowels.contains(String.valueOf(ch))){
                vowelCount++;
            } else if (Character.isLetter(ch)) {
                consonantCount++;
            }
        }

        int[] counts = {vowelCount, consonantCount};
        return counts;
    }
}